code/array/array-new-vs-fixnum-times-map.rb

```diff
@@ -0,0 +1,17 @@
+require "benchmark/ips"
+
+ELEMENTS = 9
```

Comment:
@nateberkopec: It tapers off a bit, and stabilizes around 40% slower at `1_000` elements and up. *Note: logarithmic scale!*

<img width="562" alt="screen shot 2016-01-11 at 20 23 40" src="https://cloud.githubusercontent.com/assets/5259935/12233330/5da04774-b8a1-11e5-836c-a5e4e666b4a1.png">